Tensile/TensileCreateLibrary.py

```diff
@@ -1131,6 +1131,22 @@ def param(key, value):
       
       param("best-solution", True)
 
+
+def generateManifest(manifestFile: Path, contents: List[str]) -> None:
+    """Generates tensile manifest.
+
+    Generates a manifest containing all of the tensile outputs including library metadata 
+    code object files, and sources for embedding.
+    
+    Args: 
+        manifestFile: Path to file for writting manifest.
+        contents: List of items to write manifest.
+    """
+    with open(manifestFile, "w") as generatedFile:  
+      for filePath in contents:
+        generatedFile.write(f"{filePath}\n")
```

Comment:
This function seems like it's quite general and could potentially be used for creating any string delimited file. It might be best to:
1. Add an argument to this function for the delimiter
2. Update L1147 to `generatedFile.write(f"{filePath}{delimiter}")`
3. Move this function to the _Utilities_ module

In this case, it may also make sense to rename the function to a name that matches it's functionality instead of its specific objective in _TensileCreateLibrary_.

Reply:
I moved it to Utils. I wonder if Utilities and Utils should be merged.